akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala

```diff
@@ -286,7 +286,7 @@ trait ActorContext[T] extends TypedActorContext[T] {
    * @tparam Req The request protocol, what the other actor accepts
    * @tparam Res The response protocol, what the other actor sends back
    */
-  def ask[Req, Res](target: RecipientRef[Req])(createRequest: ActorRef[Res] => Req)(
+  def ask[Req, Res](target: RecipientRef[Req], createRequest: ActorRef[Res] => Req)(
```

Comment:
Doesn't that mean the Scala compiler can now no longer infer `Req` for the second parameter from the first?

Reply:
isn't that covered by some of the test code that I changed?

Reply:
Maybe it is never actually used in a way where `createRequest` needs to infer `Req`.

Reply:
Is there a use case for type inference of `Req` given that `RecipientRef` is contravariant? Usually, you already state which message to send. The type that needs to be inferred in most cases is `Res` and that should still work.